internal/native/client.go

```diff
@@ -323,6 +323,61 @@ func (c *Client) PutContainer(params map[string]string) PutContainerResponse {
 	return PutContainerResponse{Success: true, ContainerID: contID.EncodeToString()}
 }
 
+type filter struct {
+	Key       string
+	Operation string
+	Value     string
+}
+
+func (c *Client) Search(cnrString string, filtersJS []filter) (int, error) {
```

Comment:
lets add docs at least to know what does `int` return means

Reply:
this doc is not expected to be read by anyone. this code is also impossible to use by a human, only as an extension inside a virtual machine

there are some well-known extensions that are "documented" via examples: https://github.com/grafana/xk6-loki/tree/5248b47b4efdd234403ba13e6f46dc173f9a142e/examples or via a readmi file (https://github.com/grafana/xk6-loki/blob/5248b47b4efdd234403ba13e6f46dc173f9a142e/README.md)

not sure if it applies to us (#85)

Reply:
is this an autogen code? If not, any dev can read and support it. Small tip would be appreciated

Reply:
it is not a problem to me, added a comment. now we have one documented method out of 7